students/aminavi/Lightning.py

```diff
@@ -0,0 +1,34 @@
+import random
+import datetime
+
+names = '''
+Arasim  David
+Chen    Qi
+Cheung  Jackie
+Degraw  Shawn D
+Guo Ying
+Jakubiak    Jason
+Meese   Tim
+Minavi  Alexander
+Morrison    Isaac
+Nalla   Arun K
+Nguyen  Brandon Q
+Schirra Rachel Elizabeth
+Song    Yushu
+Tetteh  Laud N
```

Comment:
Oops, this is not good, to show all names. Please remove this file from your directory. You already have this file in the `examples` directory.